### PR TITLE
Closes #1498: Update eslint and prettier

### LINKED ISSUE #1498: Remove eslint errors when using latest.
Our current settings for eslint 
https://github.com/az-digital/az_quickstart/blob/9ab40be808081577826645f01fc88cc6c93f2066/.eslintrc.json#L28-L32

Quickstart uses these eslint plugins: https://github.com/az-digital/az_quickstart/blob/main/package.json#L25-L32

Drupal core uses  totally different versions https://git.drupalcode.org/project/drupal/-/blob/9.4.x/core/package.json#L76-82

We should consider updating our package.json to match what drupal core uses, but in the meantime we can just remove the settings array from our eslintrc

### PR DESCRIPTION
**Pull request by trackleft**
## Description
Matched drupal core eslint and prettier config.

## Related issues
Closes #1498 

## How to test
Probo

## Types of changes

Coding standards updates.

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
- Minor release changes
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- [ ] Other or unknown

### Drupal core
- Patch release changes
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major or minor level update
- [x] Other or unknown

### Drupal contrib projects
- Patch release changes
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major level update
- [ ] Other or unknown

## Checklist
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by joeparsons — DISMISSED** [review]
Looks good to me.  Thanks for working on this.

So has prettier been enforcing double quotes in JS for us all this time just because we were missing that prettier config file? 😞

**Comment by trackleft** [comment]
> Looks good to me. Thanks for working on this.
> So has prettier been enforcing double quotes in JS for us all this time just because we were missing that prettier config file? 😞

Yeah, haha

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+modules/locale/tests/locale_test.es6.js
```
Comment (trackleft) [inline .prettierignore:1]:
```suggestion
```

Comment (trackleft) [reply .prettierignore:1]: Oops, we don't need that line.

**Review by jwmoore1 — APPROVED** [review]

File: .prettierignore (hunk 1 above)
Comment (joeparsons) [reply .prettierignore:1]: Do you want to commit this change so we can re-approve?

**Review by joeparsons — APPROVED** [review]

**Review by jwmoore1 — APPROVED** [review]

**Comment by joeparsons** [comment]
This will need to be updated with main again after #1462 is merged.

**Pull request merged**